src/layouts/Menu.tsx

```diff
@@ -90,7 +91,8 @@ const Menu: React.FC<{}> = React.memo((props) => {
     window.Metamask.getOraiBalance(
       metamaskAddress,
       undefined,
-      infoEvm?.rpc ?? BSC_RPC
+      infoEvm?.rpc ?? BSC_RPC,
+      BEP20_ORAI
```

Comment:
If we only put BEP20_ORAI as denom parameter for `getOraiBalance` then do we really need to set `denom' as param?

Reply:
I think i will change them to match both ```metamask``` and ```owallet```.